File: models/forum.go
package models

type Forum struct {
	Title   string
	Slug    string
	User    string
	Threads int
	Posts   int
}

type Forum1 struct {
	Posts   int    `json:"posts"`
	Slug    string `json:"slug"`
	Threads int    `json:"threads"`
	Title   string `json:"title"`
	User    string `json:"user"`
}

type ForumInput struct {
	Slug  string `json:"slug"`
	Title string `json:"title"`
	User  string `json:"user"`
}

File: frontend/src/components/forum/Forum.js
import React from "react";
import './Forum.css'
import {Link} from "react-router-dom";

const Forum = (props) => {
  const {title, user, threads, posts, slug} = props.forum;
  return (
    <div className={"forum"}>
      <div className={'forum__header'}>
        <Link to={'/forum/' + slug} className={'forum__title forums__first-col'}>
          {title}
        </Link>
        <div className={'forum__user'}>
          Author: {user}
        </div>
      </div>
      <div className={'forum__numeric-info forums__col'}>
        {threads}
      </div>
      <div className={'forum__numeric-info forums__col'}>
        {posts}
      </div>
    </div>
  )
};

export default Forum;